detection_rules/schema.py

```diff
@@ -127,13 +146,17 @@ class SiemRuleApiSchema(jsl.Document):
     # output_index = jsl.StringField(required=False)  # this is NOT allowed!
     references = jsl.ArrayField(jsl.StringField(), required=False)
     risk_score = jsl.IntField(minimum=0, maximum=100, required=True, default=21)
+    risk_score_mapping = jsl.ArrayField(jsl.DocumentField(RiskScoreMapping), required=False, min_items=1)
     rule_id = jsl.StringField(pattern=UUID_PATTERN, required=True)
+    rule_name_override = jsl.StringField(required=False)
```

Comment:
Initial support is for specifying a single field to map to `signal.rule.name`, but would be nice to support templating here, thoughts?

Reply:
Templating derived from various fields would be great. 

```
<process.name> executed the command <process.command_line> on <host.name>
```